bucket/asciidoctorj.json

```diff
@@ -0,0 +1,27 @@
+{
+    "description": "AsciidoctorJ -  Asciidoctor integration & API for Java and other JVM languages",
```

Comment:
```suggestion
    "description": "Asciidoctor integration & API for Java and other JVM languages",
```

Reply:
I've updated the description to your suggestion